data/json/items/gunmod/muzzle.json

```diff
@@ -90,15 +88,14 @@
     "location": "muzzle",
     "mod_targets": [ "pistol", "smg", "rifle" ],
     "install_time": 200,
-    "damage_modifier": -4,
     "loudness_modifier": -50,
     "recoil_modifier": -25
```

Comment:
It's not free. They consume sufficient space to make the gun not fit in a holster and are relatively rare. The former is a significant disadvantage. Install time is just to stop shenanigans like draw from holster then attaching suppressor at 0 moves cost